src/components/Search/SearchContact.js

```diff
@@ -73,7 +73,7 @@ const styles = StyleSheet.create({
     zIndex: 10,
     padding: 7,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.lightgrey
+    borderBottomColor: COLORS.gray100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the change from `COLORS.lightgrey` to `COLORS.gray100` for the `borderBottomColor` property is acceptable if it fits into your new color scheme. Again, ensure that this change does not negatively impact the visibility of the border due to contrast issues with adjacent colors.
```diff
-    borderBottomColor: COLORS.lightgrey
+    borderBottomColor: COLORS.gray100
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->